node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/yum/Yum.java

```diff
@@ -118,9 +118,9 @@ public boolean installFixedVersion(TaskContext context, YumPackageName yumPackag
         //       - "Nothing to do"
         //     And in case we need to downgrade and return true from converge()
 
-        CommandLine commandLine = terminal
-                .newCommandLine(context)
-                .add("yum", "install", "--assumeyes", yumPackage.toName());
+        CommandLine commandLine = terminal.newCommandLine(context).add("yum", "install");
+        for (String repo : repos) commandLine.add("--enablerepo=", repo);
```

Comment:
This will result in tokens like
"yum" "install" "--enablerepo=" "non-core" "--asumeyes" "nano"
which it parses as enable repo empty string, and `non-core` as yum command.

Reply:
These repos also need to be handled when downgrade is required (Line 131)